package.json

```diff
@@ -36,6 +36,7 @@
     "start-clock": "node --max_old_space_size=1500 --nouse-idle-notification clock/index.js",
     "start-worker": "node --max_old_space_size=1500 --nouse-idle-notification worker/jobProcessor.js",
     "test-api": "mocha -R dot --recursive tests/api",
+    "test-api-log": "ENABLE_API_LOGS=true mocha -R dot --recursive tests/api",
```

Comment:
@iamigo Another way of testing would be to toggleOverride enable api logs for all test api tests, but how would that differ from what is happening here?
As for how running this line not being counted by coveralls, as long as coveralls counts the original tests/api, we don't need coveralls to count it again.